Digipost/src/main/res/layout/activity_login.xml

```diff
@@ -19,126 +19,222 @@
         xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
layout for innloggingssiden er drastisk endret, og nye farger er satt. Se bilde: <img width="433" alt="Screenshot 2019-06-19 at 14 15 42" src="https://user-images.githubusercontent.com/6074094/59838638-900d4900-934f-11e9-9e4f-979d1b44b233.png">